itez/templates/beneficiary/beneficiary_list.html

```diff
@@ -62,6 +65,7 @@
                         <div class="card-body">
                           <div class="d-sm-flex justify-content-between align-items-start">
                             <div>
+                              {% include 'includes/modal.html' %}
                               <h4 class="card-title card-title-dash">All Beneficiaries</h4>
                               <p class="card-subtitle card-subtitle-dash">50+ new beneficicary registrations today</p>
```

Comment:
lets add a count of all beneficiaries registered for the curreent day here 
i.e
`50+ new beneficicary registrations today`
to say 
`{{ total_registered_today }} new beneficicary registrations today`

Reply:
I added this as well.